includes/abstracts/abstract-wc-order.php

```diff
@@ -1633,6 +1633,7 @@ public function update_taxes() {
 				continue;
 			}
 			$saved_rate_ids[] = $tax->get_rate_id();
+			$tax->set_rate( $tax->get_rate_id() );
```

Comment:
unintended change?

Reply:
Definitely unintended. Not sure where it came from apologies.